README.md

```diff
@@ -247,12 +247,15 @@ You can also see `examples/scripts` to have a feeling of how to script a Thing.
 
 #### Protocol Support
 
-* HTTP :heavy_check_mark:
-* HTTPS :heavy_check_mark:
-* CoAP :heavy_check_mark:
-* CoAPS :heavy_check_mark:
+* [HTTP](https://github.com/eclipse/thingweb.node-wot/blob/master/packages/binding-http/README.md) :heavy_check_mark:
+* [HTTPS](https://github.com/eclipse/thingweb.node-wot/blob/master/packages/binding-http/README.md) :heavy_check_mark:
+* [CoAP](https://github.com/eclipse/thingweb.node-wot/blob/master/packages/binding-coap/README.md) :heavy_check_mark:
+* [CoAPS](https://github.com/eclipse/thingweb.node-wot/blob/master/packages/binding-coap/README.md) :heavy_check_mark:
 * Websocket :heavy_check_mark:
-* MQTT :heavy_check_mark:
+* [MQTT](https://github.com/eclipse/thingweb.node-wot/blob/master/packages/binding-mqtt/README.md) :heavy_check_mark:
+* [OPC-UA](https://github.com/eclipse/thingweb.node-wot/blob/master/packages/binding-opcua/README.md) :heavy_check_mark:
+* [NETCONF](https://github.com/eclipse/thingweb.node-wot/blob/master/packages/binding-netconf/README.md) :heavy_check_mark:
+* Modbus :heavy_multiplication_x (wip)
```

Comment:
propose again using + showing that something is missing

:heavy_multiplication_x --> :heavy_plus_sign: